bulbs/content/models.py

```diff
@@ -117,6 +117,9 @@ def __unicode__(self):
         """
         return self.name
 
+    def is_new(self):
```

Comment:
just a preference, but I like to make these kinds of methods properties.

```
@property
def is_new(self):
    return self.pk is None
```

referenced as follows
```
bool(instance.is_new) "True"
```